core/src/test/java/org/infinispan/distribution/rehash/NonTxPutIfAbsentDuringJoinStressTest.java

```diff
@@ -24,7 +24,8 @@
  *
  * @author Dan Berindei
  */
-@Test(groups = "functional", testName = "distribution.rehash.NonTxPutIfAbsentDuringJoinStressTest")
+//unstable. test fails with DIST_SYNC and SCATTERED_SYNC (ISPN-3918)
+@Test(groups = "unstable", testName = "distribution.rehash.NonTxPutIfAbsentDuringJoinStressTest")
```

Comment:
IIRC you can leave the functional group, as in `groups = {"functional", "unstable"}`